libafl/src/events/llmp.rs

```diff
@@ -892,6 +892,25 @@ where
     }
 }
 
+/// Specify if the State must be persistent over restarts
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum LlmpSaveState {
+    /// Always save and restore the state (not OOM resistant)
+    Always,
```

Comment:
I feel like Always sounds weaker than OOM but actually OOM is the "stronger" state save, right? Maybe `OnRestart`?

Reply:
If I name it LllmpTrySaveState it works, maybe OOM is a bit unclear but dunno how to call it, AlwaysButOOM? Ugly

Reply:
Why not `OnRestart`? That's when it gets saved, right? Else just `Default`?
And then the OOM could be `OnRestartOOMSafe`/`OnRestartOutOfMemorySafe` or just `Always`